linux/instructions/step_5.md

```diff
@@ -0,0 +1,11 @@
+Step 4: Git branch management
+==============================
+
+We just got hired by a very famous movie star (who's name may not be disclosed)
```

Comment:
Let's find a way to tie this into the fact that they're on the box already, and it's not a "new" customer etc.